docs-zh/guide/getting-started.md

```diff
@@ -110,18 +110,55 @@ def build():
 - 环境更新。你是否总是需要问基础设施工程师如何在 Dockerfile 中添加一个新的 Python/R/Julia 包？
 - 管理环境和机器。你是否总是忘记哪些机器是用于特定项目的，因为你同时处理多个项目？
 
-## 试试 envd
+## 三分钟建立你的 envd 环境
 
 ### 安装要求
 
 - Docker (20.10.0 or above)
 
 ### 安装和初始化 `envd`
 
-`envd` 可以用 `pip` 来安装（只支持 Python3）。安装完成后，请运行 `envd bootstrap` 来初始化。
+<tabs>
+<tab name="安装脚本">
+
+在终端中运行以下命令即可安装最新版本的 `envd`:
+
+```bash
+curl -sSfL https://envd.tensorchord.ai/install.sh | sudo sh
+```
+
+</tab>
+<tab name="Homebrew">
+
+如果你使用的是 MacOS，可以通过 [homebrew](https://brew.sh/) 来安装 `envd`。
```

Comment:
Should we add a github action as https://github.com/Homebrew/homebrew-core/pull/118130#discussion_r1048501538 said?

Reply:
No, I think most package maintainers don't do that. The homebrew maintainers are very active and there is a bot @BrewTestBot doing that.

Reply:
Then, for example, we already had a new release v0.2.6. How do we update it in homebrew?

Reply:
We can wait for half a day to see if someone has submitted an update PR 😆 .  And we'll see whether we need to setup the update workflow ourselves.

Reply:
SGTM!

Reply:
No worry, look at the PR list, most of them come from maintainers or bots. I have never updated the `pdm` formula myself.